### ISSUE TITLE
Breaking semver change introduced in v4.3.0

### ISSUE THREAD

**Issue by cahamiltonAT** [styled-components/polished#644]
- `polished` 4.3.0
- `JSS-in_CSS` library and version: N/A
- Any relevant JS library and version: N/A

## Mixin/Helper/Shorthand Usage

N/A

## What You Are Seeing

```stdout
60.80 error polished@4.3.0: The engine "node" is incompatible with this module. Expected version ">=20". Got "18.19.0"
60.81 error Found incompatible module.
```

https://github.com/styled-components/polished/compare/v4.2.2...v4.3.0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R127

## Reproduction

Install v4.3.0 on any Node versions previously supported by v4.2.2

**Comment by omerkucuk-teknasyon** [comment]
Same problem for us...

**Comment by YogiDhingani27** [comment]
We're also facing the same issue.

![image](https://github.com/styled-components/polished/assets/94128664/270d033a-9fb6-4852-80bb-fab513935a4a)

**Comment by smartsahil** [comment]
Same problem here .....

**Comment by bhough** [comment]
Apologies for the error here, we have deprecated 4.3.0 and 4.2.2 should be properly distributing as the latest again on npm. We will follow up with a proper sem-ver release shortly. 

Leaving this open for others running into the issue.

**Comment by smartsahil** [comment]
It is working fine now. Thank you for your support.


Kind Regards,

Sahil Kanjariya

Software Developer
smartSense Consulting Solutions Pvt Ltd

Website:   www.smartsensesolutions.com



smartSense: *“Grow Your Business Smartly”*

*This is an eco-friendly email. Please keep printing to a minimum.*

**Comment by bhough** [comment]
This has been addressed and we've released the intended non-breaking updates in #647.
